native/dotnet-sdk-8.0/digests

```diff
@@ -0,0 +1,3 @@
+dotnet-sdk-8.0.301-linux-x64.tar.gz SHA1 c6a978b16e2462ce91446d987cca349c875db5d3
+dotnet-sdk-8.0.301-linux-x64.tar.gz SHA256 d237cf0e8865f51eb9634df32a8bcfb1dd95dac95c0d16b5d599df867ab16c27
+dotnet-sdk-8.0.301-linux-x64.tar.gz MD5 d91e94323d8db5551c9a5179162f2532
```

Comment:
please use "make digests" to create the digests file
and/or change your editor settings to keep the trailing linefeed.